netmq/HelloWorld.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NetMQ;
+using NUnit.Framework;
+
+namespace netmq
+{
+    /// <summary>
+    /// Problem: We want to send a message and receive a reply
+    /// Solution: Let's write a simple "Hello World" client and server.
+    /// </summary>
+    [TestFixture]
+    public class HelloWorld
+    {
+        [Test]
+        public void HelloWorldTest()
+        {
+            // context must be created to create sockets, one context per application
+            using (NetMQContext context = NetMQContext.Create())
+            {
+                Task.Factory.StartNew(() => Server(context));
+                Client(context);
+            }                
+        }
+
+        /// <summary>
+        /// Client does the following:
+        /// * Creates a Request socket
+        /// * Sends a "Hello" message
+        /// * Waits for a "World" reply
+        /// </summary>
+        /// <param name="context"></param>
+        private void Client(NetMQContext context)
+        {
+            // creating the request socket inside using to automatically dispose the socket
+            using (var request = context.CreateRequestSocket())
+            {
+                // connecting to the response socket
+                request.Connect("tcp://localhost:5555");
+
+                // sending a requst message, SendFrame was mulitple overloading you can use
```

Comment:
Some typos here: "Sending a request message, SendFrame has multiple overload you can use".